webroot/results/admin/persons_finish_unfinished_ACTION.php

```diff
@@ -125,6 +141,30 @@ function insertPerson( $newName, $newCountry, $newId ) {
   
   #--- Execute the command.
   dbCommand( $command );
+
+  if ( $oldCountry <> $newCountry || $oldName <> $newName){
+
+    #--- Guess countries iso2
+    $newCountryIso2 = getIso2FromCountryId( $newCountry );
+    if ( $newCountryIso2 ){
```

Comment:
Just checking if I understand this correctly... this is error checking to make sure that `getIso2FromCountryId()` did not run into trouble?